[2017/04/25 16:27:36] :: [pool-1-thread-3] :: Base path for the test = C:\New Selenium\Salesforce_QA_Automation\ADT_Resources\<br><br>[2017/04/25 16:27:36] :: [pool-1-thread-3] :: Current Thread Name for the test is "pool-1-thread-3"<br><br>[2017/04/25 16:28:33] :: [pool-1-thread-3] :: The user has launched 'http://test.salesforce.com<br><br>[2017/04/25 16:28:34] :: [pool-1-thread-3] :: The user has creating WaitListorder<br><br>[2017/04/25 16:28:57] :: [pool-1-thread-3] :: arindom.das@move.com.qa has successfully loggedin<br><br>[2017/04/25 16:29:01] :: [pool-1-thread-3] :: Successfully navigated to Home page<br><br>[2017/04/25 16:29:02] :: [pool-1-thread-3] :: File path is = C:\New Selenium\Salesforce_QA_Automation\ADT_Resources\ .<br><br>[2017/04/25 16:29:20] :: [pool-1-thread-3] :: Successfully New order account ID Fetched= readSFInputData .<br><br>[2017/04/25 16:29:30] :: [pool-1-thread-3] :: Successfully Account ID is visible= 001j000000YqQsU .<br><br>[2017/04/25 16:29:59] :: [pool-1-thread-3] :: Successfully Account Name is visible 001j000000YqQsU .<br><br>[2017/04/25 16:30:34] :: [pool-1-thread-3] :: Successfully captured card number,Card number=1111,Amount=100.0<br><br>[2017/04/25 16:30:56] :: [pool-1-thread-3] :: Successfully clicked on RenewPreauth .<br><br>[2017/04/25 16:31:22] :: [pool-1-thread-3] :: Successfully Entered preauth date<br><br>[2017/04/25 16:31:29] :: [pool-1-thread-3] :: Successfully Entered CreditCardLstNum1111 .<br><br>[2017/04/25 16:31:41] :: [pool-1-thread-3] :: Successfully clicked on WaitlistSubmit .<br><br>[2017/04/25 16:33:00] :: [pool-1-thread-3] :: Successfully clicked on larget waitlist number<br><br>[2017/04/25 16:33:47] :: [pool-1-thread-3] :: Successfully edited and saved Docusign status<br><br>[2017/04/25 16:33:58] :: [pool-1-thread-3] :: Successfully captured Waitlis name,WaitList=W-7925<br><br>[2017/04/25 16:34:01] :: [pool-1-thread-3] :: Wait list staus is renewed<br><br>[2017/04/25 16:34:02] :: [pool-1-thread-3] :: Exception occurred. C:\New Selenium\Salesforce_QA_Automation\ADT_Resources\OutPutData.xlsx (The process cannot access the file because it is being used by another process)<br><br>[2017/04/25 16:34:08] :: [pool-1-thread-3] :: Closing the Browser Instance<br><br>[2017/04/25 16:34:09] :: [pool-1-thread-3] ::  File Name: Assert.java Method name: fail(91)<br> File Name: Assert.java Method name: assertTrue(43)<br> File Name: Assert.java Method name: assertTrue(54)<br> File Name: Wait_list.java Method name: editingDocusignStatusForRenewWaitlist(1058)<br> File Name: C:/New Selenium/Salesforce_QA_Automation/ADT_Resources/SalesForce/FeatureFiles/RENEWAL_WAITLIST/Copy (4) of Renew_Pre_Auth_Waitlist.feature Method name: Then I am editing docusign status for renew waitlist(12)<br><br>Complete Error Message: <br><br><br>